zero/core/utils.py

```diff
@@ -275,7 +275,7 @@ def run_docker_container(
             command=command,
             image=str(image[0]),
             device_requests=device_requests,
-            detach=True,
+            detach=False,
```

Comment:
based on the offline discussions, I assume this is the way to have access to the running container for errors and other outputs??
1. if so, can u elaborate on this a little? 
2. also, how are we now handling the outputs from the container? does it automatically return errors and raise exceptions? 
3. if so, how does the cli respond to it?
- lets confirm that the cli Aborts as expected when the docker raises exceptions or returns failures.
- the outputs from the docker are well casted and viewed on the terminal. (catching exceptions, outputs, converting bstrings to correct strings for outputs, etc.

Reply:
> also, how are we now handling the outputs from the container? does it automatically return errors and raise exceptions?

It raises exceptions in the zero.log file 


> if so, how does the cli respond to it?

cli stops and shows aborted.

Reply:
I don't have any such references... removing it worked so I did it

Reply:
Let's have this as an issue. For I can't find anything as of now. We can have it as an issue to keep it as an reminder.

Reply:
sure, can u raise an issue and link it here? @Abhrant 
marking resolved for now.